pkg/daemon/ceph/osd/nsenter.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package osd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+)
+
+const (
+	// mountNsPath is the default mount namespace of the host
+	mountNsPath = "/rootfs/proc/1/ns/mnt"
+	// nsenterCmd is the nsenter command
+	nsenterCmd = "nsenter"
+)
+
+var (
+	binPathsToCheck = []string{"/usr/sbin", "/sbin/"}
```

Comment:
The paths of the binaries could be different in downstream images. I know we hit some issue in the past due to this. What if instead of appending the full path, we just attempt to execute `lvm` and let it be found in the system path? Then as long as it's in the path, it will work with any downstream image. 
@BlaineEXE Thoughts on this?

Reply:
That's why I'm testing multiple paths :-). The problem with just letting `lvm` command run is that the system will try to find the binary based on the `$PATH` of the container from where we are running on. So that's not super reliable since we don't know if the container we run `nsenter` from actually has a `$PATH` set. It's a superuser binary so even downstream it's either in `/usr/sbin` or `/sbin`.

For all these reasons, I'm still in favor of keeping the loop.

Reply:
Seb's comment seems reasonable to me even if it is more work.